packages/TorneloScoresheet/src/context/AppModeStateContext.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useContext, useState } from 'react';
 import { makeUseEnterPgnState } from '../hooks/appMode/enterPgnState';
 import { makeUsePairingSelectionState } from '../hooks/appMode/pairingSelectionState';
+import { makeUseEnterTablePairingState } from '../hooks/appMode/tablePairingState';
```

Comment:
should be called useTablePairingState, not useEnterTablePairingState (the enter comes from copy pasting makeUseEnterPgn i assume)